### DISCUSSION TITLE
Playlists made of public dashboards

### DISCUSSION THREAD

**Discussion by emilyastranova** [grafana/grafana#57568] [Dashboards, Panels & Visualizations]
I was wondering if there were any plans to allow playlists to use the dashboards public URLs when publicDashboards is enabled. Having a playlist on our TV would be great, but auth is a pain with proxies and such. It would be nice if it were as easy as it is to share a public dashboard.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
